draft-ietf-rtcweb-jsep.xml

```diff
@@ -5871,12 +5871,34 @@ a=msid:-
           a copy of the packet to the "m=" line associated with that
           SSRC.</t>
 
-          <t>If the packet is an FIR (PT=PSFB, FMT=4), as defined in
-          <xref target="RFC5104" />, or an LRR (PT=PSFB, FMT=TBD), as
-          defined in <xref target="I-D.ietf-avtext-lrr" />, for each
-          FCI entry where the SSRC is found in the outgoing SSRC
-          table, deliver a copy of the RTCP packet to the "m=" line
-          associated with that SSRC.</t>
+          <t> Other feedback messages include the target SSRC in the
+          Feedback Control Information (FCI). Here, each FCI needs to
+          be processed and the SSRC field identified. Also, the
+          individual FCI, combined with the RTCP packet header context,
+          needs to be forwarded to the "m=" line associated with
+          that SSRC. Examples of this pattern are:
```

Comment:
I would change "Examples of this pattern are:" to "This includes the following:"